### PR TITLE
NOISSUE - Add /disconnect ingress and group endpoints

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

### CODE REVIEW

**Review by blokovi — DISMISSED** [review]

**Review by blokovi — APPROVED** [review]

**Review by mteodor — DISMISSED** [review]

**Review by blokovi — DISMISSED** [review]

**Review by drasko — COMMENTED** [review]
This makes sense for me, but probably @nmarcetic neds to review this one, he has more experience in NginX conf.

Additionaly, should we apply this change to an NginX conf in the core repo as well?

**Comment by manuio** [comment]
@drasko Done for the core here: https://github.com/mainflux/mainflux/pull/1528

**Review by blokovi — APPROVED** [review]

File: charts/mainflux/templates/ingress.yaml [hunk 1]
```diff
@@ -57,83 +57,27 @@ spec:
...
         paths:
-          - path: /(users?.*)
-            pathType: Prefix
-            backend:
-              service:
-                name: {{ .Release.Name }}-users
-                port: 
-                  number: {{ .Values.users.httpPort }}
-          - path: /(password?.*)
+          - path: /(users|password|tokens|groups.*)
```
Comment (mteodor) [inline charts/mainflux/templates/ingress.yaml:60]: what about group of things?

Comment (manuio) [reply charts/mainflux/templates/ingress.yaml:60]: Currently there is no redirection to things groups. We should probably do it in anither issue.

**Review by dusanb94 — APPROVED** [review]

**Pull request merged**